src/components/Conversations/Conversations.tsx

```diff
@@ -65,7 +72,48 @@ export default function Conversations() {
                 return;
             }
 
-            const listResults: MedicalScribeJobSummary[] = listHealthScribeJobsRsp.MedicalScribeJobSummaries;
+            let listResults: MedicalSoapJobSummary[] = listHealthScribeJobsRsp.MedicalScribeJobSummaries;
+
+            const jobPromises = listResults.map(async (job) => {
+                const healthSoapJobDetail = await getHealthScribeJob({
```

Comment:
Looks like there is no way to get the full list, right? as this is a POC I would say it is ok, but let's analyze performance if is required later.